.github/workflows/benches.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 env:
   VERBOSE: true
-  ENVIRONMENT_UPSTREAM: docker.pkg.github.com/timberio/vector/environment:${{ github.sha }}
+  ENVIRONMENT_UPSTREAM: timberio/ci_image:${{ github.sha }}
```

Comment:
Following up from https://github.com/timberio/vector/pull/4364/files#r499995758, why is this set to `timberio/ci_image:${{ github.sha }}` instead of `timberio/ci_image:latest`?